src/constants.ts

```diff
@@ -73,7 +73,7 @@ export const DEFAULT_COLLECT_TOKEN = IS_MAINNET
 
 // Regex
 export const URL_REGEX =
-  /^(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[\da-z]+([.\-][\da-z]+)*\.[a-z]{2,5}(:\d{1,5})?(\/.*)?$/;
+  /^(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[\da-z]+([.\-][\da-z]+)*\.[a-z]{2,63}(:\d{1,5})?(\/.*)?$/;
```

Comment:
Thanks for the contribution @lucemans 🙏🏼 

**question:** don't 63 sound huge?

Reply:
It may sound huge, but that's the maximum TLD length according to ICANN.

Reply:
Ahh that make sense, just got to know 🙏🏼 Thanks